src/package/packageVersionCreate.ts

```diff
@@ -1140,6 +1140,7 @@ export const packageXmlStringToPackageXmlJson = (rawXml: string): PackageXml =>
     isArray: (name: string) => ['types', 'members'].includes(name),
   });
 
+  // eslint-disable-next-line @typescript-eslint/no-unnecessary-type-assertion
```

Comment:
eslint showed the type assertion as unnecessary but compilation failed after 'fixing' it. Added this disablement comment instead.